src/makeCalculator.js

```diff
@@ -38,6 +38,33 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    operate(operation, number) {
+      operation.call(this, number);
+
+      return this;
+    },
+    add(number) {
+      this.result += number;
+    },
+    subtract(number) {
```

Comment:
```suggestion
    },
    
    add(number) {
      this.result += number;
    },
    
    subtract(number) {
```
Don't forget to provide empty lines between methods to make your code easier to read